speech-to-text/STT_Inference_API_Server_Colab_Notebook.ipynb

```diff
@@ -202,7 +202,7 @@
       "outputs": [],
       "source": [
         "# Attach personal token\n",
-        "!ngrok config add-authtoken <place_your_ngrok_auth_token_here>"
+        "!ngrok config add-authtoken place_your_ngrok_auth_token_here"
```

Comment:
may be good to leave the `<` in there so folks know to add it.

Reply:
That portion of the code didn't show when I committed changes the first time. Our of the page discussions led us to using a comment in the code to display that portion like we want.